### DISCUSSION TITLE
【求助】生雞肉碰到下體不會感染沙門氏菌吧

### DISCUSSION THREAD

**Discussion by EustaceSop** [share121/inter-knot#2436] [其他]
先說是幫一個兄弟代發的 不是本人
剛剛我兄弟跟我說因為家裡沒人所以他裸體 在做晚餐的時候
生雞胸肉不小心碰到下體了 應該是不會感染吧 那個兄弟很急 在線等

**Comment by MrDucker** [comment]
他最好是在做晚餐<img src="https://bbs-static.miyoushe.com/static/2023/11/16/9869e04ba6a0d0e6ed2418efc27d4b74_4411752698018709526.png" alt="苍角-诶？" width="75" >

**Comment by 127kzzz** [comment]
这又是什么play

**Comment by yytyfj** [comment]
所以你淦了生鸡胸肉？

**Comment by WFGoose** [comment]
呃~你也逆天！

**Comment by ddsgkky** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/1a2616d760a6f334a4a9abca46267af3_7281603850910753959.png" alt="三月七 我" width="75" >都是鸡应该没事吧……

**Comment by NUCKFKER** [comment]
你想干什么<img src="https://img-static.mihoyo.com/communityweb/upload/1855f119fa081eb25b742a83827ef797.png" alt="帕姆_打咩" width="75" >
